plugins/virtualization/kvm_cpu

```diff
@@ -57,9 +57,12 @@ def clean_vm_name(vm_name):
 def detect_kvm():
     ''' Check if kvm is installed
     '''
-    kvm = Popen("which kvm", shell=True, stdout=PIPE)
-    kvm.communicate()
-    return not bool(kvm.returncode)
+    if os.path.isfile("/usr/libexec/qemu-kvm"):
+        return True
+    else:
+        kvm = Popen("which kvm", shell=True, stdout=PIPE)
+        kvm.communicate()
+        return not bool(kvm.returncode)
```

Comment:
I would propose to phrase this clearer (instead of relying on the coincidence of the implicit interger/boolean conversion):
`return kvm.returncode == 0`